GliaWidgets/View/Chat/Message/Content/Choice/ChoiceCardOptionView.swift

```diff
@@ -64,6 +64,7 @@ class ChoiceCardOptionView: UIView {
             applyStyle(style.selected)
         case .disabled:
             applyStyle(style.disabled)
+            setBorderLinesForTabs()
```

Comment:
With the small refactoring I suggested, we should include the code for setting the border in `applyStyle`. If it's nil - do nothing, otherwise do what you are doing in `setBorderLinesForTabs()`